not-so-trivial-pursuit/server/index.js

```diff
@@ -1,9 +1,52 @@
 import express from "express";
+import bodyParser from "body-parser";
 import cors from "cors";
+import fetch from "node-fetch";
 
 const app = express();
-const PORT = 8080;
+const PORT = 8082;
 
 app.use(cors());
 
-app.listen(PORT, () => console.log(`Hola! Server is running on port ${PORT}`));
\ No newline at end of file
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+
+
+// app.get("/", (req, res) => {
+//   res.send("Hello Techtonica Server for a Game");
+// });
+
+
+app.get("/api/game", (req, res) => {
```

Comment:
A fun thing we could pair on would be allowing the ability to pass the category as a query parameter here! That way, if you ever built the functionality to set the category state based on the user interaction with the category buttons, you could fetch a question with the selected category

Reply:
Yes I feel like if I could have a week longer on this one I could have gotten to this point. I look forward to seeing how we incorporate these lessons into our sessions of the coming weeks.